CHANGELOG.md

```diff
@@ -76,6 +76,11 @@ Usage: `DeploymentLauncher createsim <project-name> <assembly-name> <target-depl
 - Added `OnPlayerSpawnFailed` delegate to `SpatialGameInstance`. This is helpful if you have established a successful connection but the server worker crashed. `OnConnected` and `OnConnectionFailed` are now also blueprint-assignable.
 - `Launch Deployment` in the `Cloud Deployment` dialog will now generate schema, generate a snapshot, build the server worker, build the client, build the simulate player worker, and upload the assembly before launching the deployment all in one step, there are checkboxes to toggle generation of schema and snapshots as well as whether to build the client worker.
 - You can now edit the project name field in the `Cloud Deployment` window.
+- Create dropdown menu for choosing connection flow in the Start toolbar button.
+- Move Spatial Networking checkbox to start dropdown menu.
+- Add `Connect to local deployment`, `Connect to cloud deployment`, `Do not connect` options in start dropdown menu.
+- Add `Local Deployment IP`, `Cloud Deployment Name` text editor in start dropdown menu to config the target deployment info.
+- Add `Editor Settings` field in start dropdown menu to link to the Editor settings.
```

Comment:
```suggestion
- Created a new dropdown menu in the Start toolbar button. This dropdown menu allows you to configure how to connect your PIE client or your Launch on Device client
  - You can choose between `Connect to local deployment`, `Connect to cloud deployment` and `Do not connect` to specify the flow the client should automatically take upon clicking the Play or the Launch button.
  - Added the `Local Deployment IP` field to specify which local deployment you want to connect to. By default, this will be `127.0.0.1`.
  - Added the `Cloud deployment name` field to specify which cloud deployment you want to connect to. If no cloud deployment is specified and you select `Connect to cloud deployment`, it will try to connect to the first running deployment that has the `dev_login` deployment tag.
  - Added the `Editor Settings` field to allow you to quickly get to the **SpatialOS Editor Settings**
```